src/material-experimental/popover-edit/_popover-edit-theme.scss

```diff
@@ -83,7 +82,7 @@
     display: block;
     padding: 16px 24px;
 
-    @include cdk.high-contrast(active, off) {
+    @media(forced-colors: active) {
```

Comment:
I think that we should keep around the `high-contrast` mixin, but have it use `@media` internally instead of the class. That's what we were using before the `HighContrastModeDetector` as well. This way we also don't have to migrate existing users of the mixin.

Reply:
How difficult is it to do a migration?

Reply:
Just because it's deprecated doesn't mean we have to remove it. We have the option to push back the removal date. HighContrastModeDetector will continue to work correctly because it already uses media query. Do you want to support `HighContrastModeDetector` long-term?

Reply:
I'm not saying that we should support the detector, but rather that we should keep the `high-contrast` Sass mixin around and have it generate the media query instead of the class.

Reply:
So, you agree with deprecating `HighContrastModeDetector`, but you don't want to deprecate `cdk.high-contrast` SASS mixin? Is that right?

The `high-contrast` sass mixin has a target argument that can be `active`, `white-on-black` or `black-on-white`. how do you suggest we handle `white-on-black` and `black-on-white`?

Reply:
That's right. I think that for the argument we can print out a deprecation warning.

Reply:
Okay, that sounds reasonable to me. I feel strongly that `HighContrastModeDetector` should be deprecated and we should point developers to used forced-colors. I think that will make it easier for new developers to pick up a11y.

I'm okay with keeping `cdk.high-contrast` mixing around, delaying it's deprecation, or deprecating some if it's arguments.

I'll look into updating the mixin and get back to you. I'll think of a way to have the mixin use `forced-colors` media query, but also have the same level of specificity.